frontend/integration-tests/tests/kubevirt/vm.wizard.scenario.ts

```diff
@@ -0,0 +1,101 @@
+import { execSync } from 'child_process';
+import { browser, by, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '../../protractor.conf';
+import { resourceRowsPresent, filterForName, deleteRow, isLoaded, createItemButton } from '../../views/crud.view';
+import * as vm from '../../views/kubevirt/vm.view';
+
+
+describe('Kubevirt create VM using wizard', () => {
+  const namespace = 'default';
+  const provisionSource = 'PXE';
+  const operatingSystem = 'fedora28';
+  const flavor = 'small';
+  const workloadProfile = 'generic';
+  const networkDefinitionName = `${testName}-ovs-net-1`;
+  const pxeInterface = 'eth1';
+
+  const testNAD = {
+    apiVersion: 'k8s.cni.cncf.io/v1',
+    kind: 'NetworkAttachmentDefinition',
+    metadata: {
+      name: networkDefinitionName,
+    },
+    spec: {
+      config: '{ "cniVersion": "0.3.1", "type": "ovs", "bridge": "br0" }',
+    },
+  };
+
+  beforeAll(async() => {
+    execSync(`echo '${JSON.stringify(testNAD)}' | kubectl create -f -`);
+  });
+
+  afterAll(async() => {
+    execSync(`kubectl delete net-attach-def ${networkDefinitionName}`);
+  });
+
+  it('Navigates to VMs', async() => {
+    await browser.get(`${appHost}/k8s/all-namespaces/virtualmachines`);
```

Comment:
I'm not sure whether it's working, for d/s, it's using `kubevirt-web-ui.cloud.examples.com` to access VM page currently.

Reply:
@gouyang Seems to me like you have set BRIDGE_BASE_ADDRESS env variable on the machine you run it on